Makefile

```diff
@@ -1,5 +1,20 @@
 TARGETS=go1.16 go1.17
 ARM_TARGETS=go1.16 go1.17
+# TODO: to be replaced once we validate it works as expected
+# TODO: version to be tagged too.
+#NPCAP_FILE=npcap-1.60-oem.exe
+NPCAP_FILE=test.txt
```

Comment:
This will be replaced in a follow up, once we validate everything was pushed as expected